frontend/ui/widget/inputtext.lua

```diff
@@ -671,13 +672,20 @@ end
 --- Search for a string.
 -- if start_pos not set, starts a search from the next to cursor position
 -- returns first found position or 0 if not found
-function InputText:searchString(str, start_pos)
+function InputText:searchString(str, case_sensitive, start_pos)
     local str_len = string.len(str)
     local char_pos, found = 0, 0
+    local char_txt, char_str
     start_pos = start_pos and (start_pos - 1) or self.charpos
     for i = start_pos, #self.charlist - str_len do
         for j = 1, str_len do
-            if string.lower(self.charlist[i + j]) ~= string.lower(string.sub(str, j, j)) then
+            char_txt = self.charlist[i + j]
+            char_str = string.sub(str, j, j)
+            if not case_sensitive then
+                char_txt = Utf8Proc.lowercase(char_txt)
+                char_str = Utf8Proc.lowercase(char_str)
```

Comment:
Does this `Utf8Proc.lowercase(char_str)` need to be done inside each loop ? Or can it be done before to limit the number of calls to the possibly expensive Utf8Proc.lowercase() ?